binder/src/main/java/io/grpc/binder/BindServiceFlags.java

```diff
@@ -101,6 +103,26 @@ public Builder setAdjustWithActivity(boolean newValue) {
       return setFlag(BIND_ADJUST_WITH_ACTIVITY, newValue);
     }
 
+    // TODO(b/274061424): Reference official constant and add RequiresApi declaration in place of
+    // informal Javadoc warning when U is final.
+    /**
+     * Sets or clears the {@code android.content.Context#BIND_ALLOW_ACTIVITY_STARTS} flag.
+     *
+     * <p>This method allows for testing and development on Android U developer previews. Before
+     * releasing production code which depends on this flag, verify that either the
+     * {@code BIND_ALLOW_ACTIVITY_STARTS} flag has not changed from 0x200 during SDK development,
+     * or wait for this method to be updated to point to the final flag and made non-experimental.
+     *
+     * <p>This flag has no additional meaning at the gRPC layer. See the Android docs for more.
+     *
+     * @return this, for fluent construction
+     */
+    @ExperimentalApi("To be finalized after Android U SDK finalization")
+    public Builder setAllowActivityStarts(boolean newValue) {
+      // https://developer.android.com/reference/android/content/Context#BIND_ALLOW_ACTIVITY_STARTS
+      return setFlag(0x200, newValue);
```

Comment:
nit: Define a package visible constant for this? (Since it's used in the test as well).

Reply:
I was leaning against this because when the TODO is resolved, we would directly use the SDK constant everywhere, rather than just updating the package-visible constant. So this felt closer to the final version.

Reply:
sgtm.